src/world/area_hos/hos_05/A2A240.c

```diff
@@ -18,6 +18,30 @@ ApiStatus N(AwaitScriptComplete)(Evt* script, s32 isInitialCall) {
     }
 }
 
-INCLUDE_ASM(s32, "world/area_hos/hos_05/A2A240", func_80240634_A2A874);
+ApiStatus func_80240634_A2A874(Evt* script, s32 isInitialCall) {
```

Comment:
in the future, try to N()-ify decomped map funcs

Reply:
I disagree. it's easy to do in one go later on, and N-ifying all these symbols for the sake of it only slows decomp down and doesn't actually accomplish anything